packages/app/screens/feed/FeedScreen.tsx

```diff
@@ -10,6 +10,9 @@ import CardEvent from "@app/components/EventCard/EventCard";
 import { useUserContext } from "@app/utils/UserContext";
 import { logoutUser } from "@app/utils/datalayer";
 import { ENDPOINT } from "@app/utils/constants";
+import { EventType, ItemType } from "@app/types/EventCard";
+
+
 
 function FeedScreen() {
   const { user, setUser } = useUserContext();
```

Comment:
Getting red underline for setUser part in the "handleOnLogout" function. Maybe make data optional in State in UserContext.tsx